Apps/APN/docs/dev-notes/DEVELOPMENT.md

```diff
@@ -0,0 +1,134 @@
+# Development
```

Comment:
This file looks like it was copied from the Ami app. I would delete it from this PR.

If there are instructions inside of here that are valuable for react native sample app development, I would prefer that we have 1 README that is relevant to all of the RN sample apps such as [the one written for iOS](https://github.com/customerio/customerio-ios/blob/main/Apps/README.md).